backend/src/main/scala/cromwell/backend/ExecutionHash.scala

```diff
@@ -0,0 +1,13 @@
+package cromwell.backend
+
+import java.util.UUID
+
+import scala.concurrent.{ExecutionContext, Future}
+
+object ExecutionHash {
+  // TODO: PBE: ideally hashes should be deterministic
```

Comment:
I'm just picturing a year from now when people see all of these "TODO PBE" and wonder WTF it means. "Who were you denvercoder9? What did you see?!?"